### ISSUE TITLE
Incorrect permissions after install

### ISSUE THREAD

**Issue by eburgueno** [Cibiv/NextGenMap#38]
After successful compilation, the `ngm` script is installed with incorrect permissions under `/usr/bin`:

```
$ ls -l /usr/bin | grep ngm
-rwxr-x---. 1 root root        466 May 23 14:16 ngm
-rwxrwxr-x. 1 root root   15618416 May 23 14:16 ngm-core
-rwxrwxr-x. 1 root root   14665096 May 23 14:16 ngm-core-debug
-rwxr-x---. 1 root root        483 May 23 14:16 ngm-debug
-rwxr-x---. 1 root root        472 May 23 14:16 ngm-log
-rwxrwxr-x. 1 root root   15054712 May 23 14:16 ngm-utils
-rwxrwxr-x. 1 root root   11280904 May 23 14:16 ngm-utils-debug
```

These should all have 755 permissions, so that all users on the system can run them.

It also affects the Docker container when run as a non-privileged user with `docker run -u 1000:1000 -v etc`. Singularity containers using the same set of installation instructions fail for the same reason.

**Comment by fritzsedlazeck** [comment]
Did you compile it from the source code?

**Comment by eburgueno** [comment]
Yes, this is when manually building the Docker container from your [Dockerfile](https://github.com/Cibiv/NextGenMap/blob/master/docker/Dockerfile). But the steps are the same.

**Comment by mbhall88** [comment]
I am also having a similar issue. I am building `ngm` in a singularity container but am unable to run `ngm` from outside the container due to permissions issues.
